packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -235,12 +250,16 @@ export default compose( [
 			getSettings,
 		} = select( 'core/block-editor' );
 
+		const horizontalDirection =
```

Comment:
Let's add it to the returned props here and then use it here https://github.com/WordPress/gutenberg/pull/19013/files#diff-260b0f5b413f5ac735a6935b2f8acc34R213

Reply:
thanks, will apply that one as well.

I just wonder if we shouldn't also add `__experimental` prefix and also change the name to `isHorizontalDirection` so finally the name will be `____experimentalIsHorizontalDirection` because we base on `__experimentalMoverDirection`  when passing this prop.

Or alternatively pass `__experimentalMoverDirection` and add some selector which compare this prop to `horizontal` each time we need this.

```jsx

getHorizontalDirection( __experimentalMoverDirection ) =>  __experimentalMoverDirection === 'horizontal';

```

WDYT ? @pinarol  @dratwas